src/averageAges.js

```diff
@@ -36,8 +44,20 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = false) {
```

Comment:
```suggestion
function calculateWomenAverageAge(people, withChildren) {
```
it would be undefined, if not passed